blogs/big_data/data_visualization/blog.md

```diff
@@ -0,0 +1,252 @@
+
+![](https://i.imgur.com/KUUBE5R.jpg)
+
+Select California, Illinois, New Jersey, and New York, which are the states with most coronavirus cases.
+
+![](https://i.imgur.com/osQETRU.jpg)
+
+Tada!
+The graph below displays COVID-19 cases by state.
+
+![](https://i.imgur.com/0HDFtwb.jpg)
+
+If you change the date range, the bar graph updates accordingly.
+
+![](https://i.imgur.com/schpEg4.jpg)
+
+If you would like to apply the date range filter to sheet 1 (COVID-19 case by state and County), you can do so by applying the date filter to all worksheets using the same data source. Now, the date filter will be applied to both sheet 1 and sheet 2.
+
+![](https://i.imgur.com/3XZuNIM.jpg)
+
+Step 4: Format the dashboard
+- 
+
+In dashboard 1, drag sheet 1 to the blank page.
+
+![](https://i.imgur.com/ORdHKDE.jpg)
+
+Then you will notice that this dashboard does not have any space to display the bar graph we created in sheet 2. We should expand the size of the dashboard. 
+
+The default size is 1000 x 800. Let’s increase the height to 1600.
+
+![](https://i.imgur.com/iLXt34y.jpg)
+
+
+Select sheet 1 and click the downward arrow and select Floating. This allows you to freely move the table and resize the table.
+
+![](https://i.imgur.com/K25QOae.jpg)
+
+Click “Floating” on State and County filter as well and move them around as you would like.
+
+![](https://i.imgur.com/eAg9Add.jpg)
+
+Follow the same steps above and add sheet 2 to the dashboard.
+
+![](https://i.imgur.com/NGeWYvY.jpg)
+
+Now, let's add title to the dashboard!
+
+Select "Show dashboard title" in the Objects section and add the title.
+
+![](https://i.imgur.com/hbZbZ1h.jpg)
+
+The dashboard looks too plain, so let's add a background color to the title.
+
+![](https://i.imgur.com/8Er750I.jpg)
+
+Right click the title and click "Format title…"
+
+![](https://i.imgur.com/YDKw8AU.jpg)
+
+Apply Shading in the Dashboard Title section.
+
+![](https://i.imgur.com/RsmLOhP.jpg)
+
+
+Here is the final product of the dashboard!
+
+![](https://i.imgur.com/dbwHZVE.jpg)
+
+![](https://i.imgur.com/4Y8RT3m.jpg)
+
+Feel free to check out the dashboard we just created! Play around with the date range filter, state and county filter and see what happens!
+
+https://public.tableau.com/profile/narae.lee#!/vizhome/COVID-19StatusDashboardatCountyLevel/Dashboard1?publish=yes
+
+
+In this blog, we have learned how to connect to a dataset, create a table and a line graph, and add different types of filters. As these features are the basics of data visualization, you can apply these features in any Tableau dashboard. 
+
+If you would like to dive into Tableau further, I would recommend working with a more complicated dataset and looking into the following features:
+
+1. Create a calculated field, which allows you to create a new column from data that already exists in your dataset
+2. Work with maps in Tableau
+
+Thank you for reading my blog!
```

Comment:
I would eliminate this last sentence — it's nice, but I think it would be better to end on your recommendations for more complicated datasets